okapi/services/caches/geocaches.php

```diff
@@ -724,10 +739,6 @@ public static function call(OkapiRequest $request)
                     $tmp[$row['cache_id']][] = $row;
                 foreach ($tmp as $cache_key => &$rowslist_ref)
                 {
-                    usort($rowslist_ref, function($rowa, $rowb) {
-                        # (reverse order by date)
-                        return ($rowa['date'] < $rowb['date']) ? 1 : (($rowa['date'] == $rowb['date']) ? 0 : -1);
-                    });
```

Comment:
Huh. Indeed. Seems completely redundant. Must have missed it during some refactoring...